crates/viewer/re_viewer/src/app.rs

```diff
@@ -1575,6 +1575,8 @@ impl App {
                     self.show_text_logs_as_notifications();
                 }
             });
+
+        self.notifications.show_toasts(egui_ctx);
```

Comment:
This was previously called from the same function that showed the notification button, which I think was a mistake. Even if the top panel is hidden we should still show the toast notifications.